sdk/core/azure-core/azure/core/pipeline/policies/_universal.py

```diff
@@ -104,6 +104,56 @@ def on_request(self, request):
         if additional_headers:
             request.http_request.headers.update(additional_headers)
 
+class RequestIdPolicy(SansIOHTTPPolicy):
+    """A simple policy that sets the given request id in the header.
+
+    This will overwrite request id that is already defined in the request. Request id can be
+    configured up front, where the request id will be applied to all outgoing
+    operations, and additional request id can also be set dynamically per operation.
+
+    :param dict base_headers: Headers to send with the request.
+
+    .. admonition:: Example:
+
+        .. literalinclude:: ../samples/test_example_sansio.py
+            :start-after: [START request_id_policy]
+            :end-before: [END request_id_policy]
+            :language: python
+            :dedent: 4
+            :caption: Configuring a request id policy.
+    """
+    def __init__(self, request_id=None, **kwargs):  # pylint: disable=super-init-not-called
+        # type: (str, dict) -> None
+        self._request_id = request_id
+
+    @property
+    def request_id(self):
```

Comment:
Generally, you would only use a property in Python if you need to make it read or writeonly, or if there was logic associated with setting or getting the value. There is no need to have a simple "property that wraps an attribute" in Python. It only comes with downsides. No upside.